mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -1869,6 +1869,30 @@ func TestAccContainerCluster_withSoleTenantGroup(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withAdvancedMachineFeatures(t *testing.T) {
+	t.Parallel()
+
+	resourceName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withAdvancedMachineFeatures(resourceName, 1),
+			},
+            {
+                Config: testAccContainerCluster_withAdvancedMachineFeatures(resourceName, 2),
```

Comment:
Given update isn't supported we can omit this `Config` step, as it will just re-create the resource.

Reply:
Done!